test/NuGet.Clients.Tests/NuGet.SolutionRestoreManager.Test/VsSolutionRestoreServiceTests.cs

```diff
@@ -2183,6 +2183,49 @@ public void ToPackageSpec_TransitiveDependencyPinning_CanBeEnabled(string Centra
             }
         }
 
+        [Theory]
+        [InlineData(null, false)]
+        [InlineData("", false)]
+        [InlineData(" ", false)]
+        [InlineData("invalid", false)]
+        [InlineData("false", false)]
+        [InlineData("true", true)]
+        [InlineData("           true    ", true)]
```

Comment:
I tested a few other MSBuild properties with padded spaces, and as I expected, got build errors. Do we really want to accept spaces in this particular property? Is it just to try to be friendly to typos, or is there a specific reason?